src/util/vec/sse2.hpp

```diff
@@ -20,29 +20,29 @@ struct f128 {
 #endif
 
     // ---- Constructors ----
-    static inline f128 make(double a, double b) {
+    static inline f64x2 make(double a, double b) {
 #if F128_USE_SSE2
-        f128 r;
+        f64x2 r;
         r.v = _mm_set_pd(b, a);
         return r;
 #else
         return {a, b};
 #endif
     }
 
-    static inline f128 broadcast(double x) {
+    static inline f64x2 broadcast(double x) {
 #if F128_USE_SSE2
-        f128 r;
+        f64x2 r;
         r.v = _mm_set1_pd(x);
         return r;
 #else
         return {x, x};
 #endif
     }
 
-    static inline f128 zero() {
+    static inline f64x2 zero() {
 #if F128_USE_SSE2
```

Comment:
```suggestion
#if F64x2_USE_SSE2
```

we're using f64x2 naming right?